clrdefinitions.cmake

```diff
@@ -7,7 +7,7 @@ endif()
 if (WIN32)
   set(FEATURE_EVENT_TRACE 1)
 endif()
-if(CLR_CMAKE_PLATFORM_LINUX AND CLR_CMAKE_PLATFORM_ARCH_AMD64)
+if(CLR_CMAKE_PLATFORM_LINUX AND CLR_CMAKE_TARGET_ARCH_AMD64)
   set(FEATURE_EVENT_TRACE 1)
```

Comment:
This is enabling Eventing on Linux, we have only explored this feature for AMD64.
I think Arm64 for linux is still work in progress. And for arm there is no native eventing story yet